### ISSUE TITLE
Add option to ignore SSL certificate

### ISSUE THREAD

**Issue by Nklya** [rrreeeyyy/exporter_proxy#8]
Hello.
Thank you for this tool!

I think it would be nice to add an option to ignore the SSL certificate from the exporter.

Use case:
I want to collect metrics from a local application which use TLS. When I point exporter_proxy to localhost, I get an error with ssl validation.
So I have to use fqdn.

**Comment by Nklya** [comment]
Thank you!

**Comment by rrreeeyyy** [comment]
Hi @Nklya .

I understand your use case.
I implemented `insecure_skip_verify` configuration to disable validation of the server certificate in #9.
I released it as v0.4.0. Could you check it?

**Comment by Nklya** [comment]
Hi.

I tested and it works, but I use chef to provision exporter_proxy config and I get an error because of using quotes around bool value.

```
Jun 25 12:08:52 streamer exporter_proxy[21474]: 2019/06/25 12:08:52 yaml: unmarshal errors:
Jun 25 12:08:52 streamer exporter_proxy[21474]:   line 17: cannot unmarshal !!str `true` into bool
```

So this works - `insecure_skip_verify: true`
And this not - `insecure_skip_verify: 'true'`

I changed my Chef templates to fix it, but anyway, I think it's a good idea to handle it too.
